CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # CHANGELOG
 
+## v1.0.1
```

Comment:
This should be v1.1.0 since it's adding a new feature

Reply:
Updated change log.